debug/access_token.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 mapboxgl.accessToken = getAccessToken();
+require('dotenv').config()
```

Comment:
to access the mapbox API with my keys in a .env file, come to think of it I probably do not need to do that because I am running the server in the terminal